qiskit/aqua/algorithms/minimum_eigen_solvers/iqpe.py

```diff
@@ -193,7 +193,7 @@ def construct_circuit(self,
         qc += self._state_in.construct_circuit('circuit', q)
         # hadamard on a[0]
         qc.add_register(a)
-        qc.u2(0, np.pi, a[0])
+        qc.u(np.pi/2, 0, np.pi, a[0])
```

Comment:
@Cryoris I recall discussions about pi in the past. I see in places we import pi from qiskit.qasm and use that. Here it was using numpy.pi - do you know if its advisable to change this at all? Since its being updated to take care of the u1-3 deprecation I just noticed here.

Reply:
`qasm.pi` is an alias for `numpy.pi`, so using `numpy.pi` should be fine 🙂

Reply:
Ok, I know in the past sympy was in the mix too.